parts/footer-alpha-v3.html

```diff
@@ -12,71 +12,67 @@
 	<div class="wp-block-group"><!-- wp:image {"width":"150px","sizeSlug":"large","linkDestination":"none"} -->
 	<figure class="wp-block-image size-large is-resized"><img src="http://digital.gov.bc.ca/wp-content/uploads/2023/02/BCID_H_rgb_pos.png" alt="" style="width:150px" title=""/></figure>
 	<!-- /wp:image -->
-	
+
 	<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|text-color"},":hover":{"color":{"text":"var:preset|color|tertiary"}}}},"spacing":{"margin":{"top":"0.15rem","bottom":"0.15rem"}}},"textColor":"text-color","className":"external","fontSize":"extra-small"} -->
-	<p class="external has-text-color-color has-text-color has-link-color has-extra-small-font-size" style="margin-top:0.15rem;margin-bottom:0.15rem">We can help in over 120 languages and through <a href="tel:+1-800-663-7867">Telephone Device For The Deaf </a>(TDD). <a href="https://www2.gov.bc.ca/gov/content/home/get-help-with-government-services">Call, email or text</a> us or find a <a href="https://www2.gov.bc.ca/gov/content/home/services-a-z">service centre</a></p>
-	<!-- /wp:paragraph --></div>
+	<p class="external has-text-color-color has-text-color has-link-color has-extra-small-font-size" style="margin-top:0.15rem;margin-bottom:0.15rem">We can help in over 220 languages and through other accessible options.
+         <a href="https://www2.gov.bc.ca/gov/content/home/get-help-with-government-services">
+            Call, email or text us
+         </a>
+         , or
+         <a href="https://www2.gov.bc.ca/gov/content/home/services-a-z">
+            find a service centre
+        </a>
+    </p> <!-- /wp:paragraph --></div>
 	<!-- /wp:group --></div>
 	<!-- /wp:column -->
-	
+
 	<!-- wp:column {"width":"15%"} -->
 	<div class="wp-block-column" style="flex-basis:15%"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"},"padding":{"top":"0","right":"0","bottom":"var:preset|spacing|20","left":"0"}},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|text-color"}}}},"textColor":"text-color"} -->
-	<h2 class="wp-block-heading has-text-color-color has-text-color has-link-color" style="margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:var(--wp--preset--spacing--20);padding-left:0;font-size:1rem">More info</h2>
+	<h2 class="wp-block-heading has-text-color-color has-text-color has-link-color" style="margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:var(--wp--preset--spacing--20);padding-left:0;font-size:1rem"><p style="font-weight: bold; font-size: 14px">MORE INFO</p>
```

Comment:
we should never put words all uppercase, stying should be used to transform words, if really required.

Reply:
Should be good now. I even made the fontsize rem